package.json

```diff
@@ -126,14 +126,19 @@
     "zone.js": "0.11.8"
   },
   "overrides": {
-    "@angular/compiler": "12.2.16",
-    "@angular/core": "12.2.16",
+    "@angular/compiler": "12.2.17",
+    "@angular/compiler-cli": "12.2.17",
+    "@angular/core": "12.2.17",
     "@types/eslint": "6.8.1",
+    "@types/node": "18.11.10",
+    "cheerio": "1.0.0-rc.10",
     "chokidar": "3.5.3",
     "fs-extra": "10.1.0",
     "glob": "7.2.3",
     "latest-version": "7.0.0",
     "selfsigned": "2.1.1",
-    "tslib": "2.4.0"
+    "tslib": "2.4.0",
+    "typescript": ">=4.2.3 <4.4",
```

Comment:
This and @types/node are the dependency changes that broke the typescript build.